projects/1.19/assets/varied-adventure/va/lang/zh_cn.json

```
@@ -0,0 +1,251 @@
+  "entity.va.death_bird": "骨鸟",
+  "entity.va.drull": "德鲁尔",
+  "entity.va.frostwing": "寒霜骨龙",
+  "entity.va.listener": "监听者",
+  "entity.va.minion_nightmare": "梦魇",
+  "entity.va.night_watcher": "守夜人",
+  "entity.va.nightmare": "梦魇",
+  "entity.va.nightmarish_observer": "梦魇观察者",
+  "entity.va.nightmarish_rune": "噩梦水晶",
+  "entity.va.patron_of_animals": "物种守护神",
+  "entity.va.poison_skull": "薨髑",
+  "entity.va.reaghtoo": "獠牙兽",
+  "entity.va.saurian": "恐蜥",
+  "entity.va.sea_monster": "海怪",
+  "entity.va.soul_rune": "灵魂水晶",
+  "entity.va.swamp_catacombs_spawner": "沼泽地下墓穴刷怪笼",
+  "entity.va.swamp_rune": "沼气水晶",
+  "entity.va.tease": "嬉笑者",
+  "entity.va.toadman": "蛙人",
+  "entity.va.unknown_thief": "无名窃贼",
+  "entity.varied_adventure.death_bird": "骨鸟",
+  "entity.varied_adventure.meadow_runner": "草原猎手",
+  "entity.varied_adventure.meadow_runner_spawner": "草原猎手刷怪笼",
+  "entity.varied_adventure.night_watchman": "守夜人",
+  "entity.varied_adventure.reaghtoo": "獠牙兽",
+  "entity.varied_adventure.reaghtoo_spawner": "獠牙兽刷怪笼",
+  "entity.varied_adventure.rocky_golem": "岩石傀儡",
+  "entity.varied_adventure.rocky_golem_spawner": "岩石傀儡刷怪笼",
+  "entity.varied_adventure.sea_monster": "海怪",
+  "entity.varied_adventure.tease": "嬉笑者",
+  "entity.varied_adventure.the_commander_of_royal_army": "皇家陆军司令",
+  "item.v_a.amber": "琥珀",
+  "item.v_a.black_feather": "黑色羽毛",
+  "item.v_a.cooked_meadow_runner_rids": "熟草原猎手肋排",
+  "item.v_a.death_bird_heart": "骨鸟之心",
+  "item.v_a.death_bird_spawn_egg": "骨鸟刷怪蛋",
+  "item.v_a.golem_vein": "傀儡脉络",
+  "item.v_a.meadow_runner_rids": "草原猎手肋排",
+  "item.v_a.meadow_runner_spawn_egg": "草原猎手刷怪蛋",
+  "item.v_a.meadow_runner_spawner_spawn_egg": "草原猎手刷怪笼刷怪蛋",
+  "item.v_a.moerolite_ingot": "莫洛利特锭",
+  "item.v_a.mysterious_crystal": "诡秘水晶",
+  "item.v_a.night_watchman_spawn_egg": "守夜人刷怪蛋",
+  "item.v_a.raw_moerolite": "粗莫洛利特矿",
+  "item.v_a.reaghtoo_spawn_egg": "獠牙兽刷怪蛋",
+  "item.v_a.reaghtoo_spawner_spawn_egg": "獠牙兽刷怪笼刷怪蛋",
+  "item.v_a.rocky_golem_spawn_egg": "岩石傀儡刷怪蛋",
+  "item.v_a.rocky_golem_spawner_spawn_egg": "岩石傀儡刷怪笼刷怪蛋",
+  "item.v_a.sea_monster_scales": "海怪鳞片",
+  "item.v_a.sea_monster_spawn_egg": "海怪刷怪蛋",
+  "item.v_a.sea_monster_tooth": "海怪尖牙",
+  "item.v_a.soul_light": "灵魂之光",
+  "item.v_a.tease_eye": "嬉笑之瞳",
+  "item.v_a.tease_spawn_egg": "嬉笑者刷怪蛋",
+  "item.v_a.the_commander_of_royal_army_spawn_egg": "皇家陆军司令刷怪蛋",
+  "item.v_a.totem_shard": "图腾碎片",
+  "item.va.amber": "琥珀",
+  "item.va.ancient_horn_sword": "古角剑",
+  "item.va.basilisk_poison_ball": "鬣蜥毒气",
+  "item.va.basilisk_skull_helmet": "巴兹里斯克蛇颅",
+  "item.va.basilisk_spawn_egg": "巴兹里斯克蛇刷怪蛋",
+  "item.va.black_feather": "黑色羽毛",
+  "item.va.cooked_reaghtoo_ribs": "熟獠牙兽肋排",
+  "item.va.copperolite_armor_boots": "铜橄榄靴子",
+  "item.va.copperolite_armor_chestplate": "铜橄榄胸甲",
+  "item.va.copperolite_armor_helmet": "铜橄榄头盔",
+  "item.va.copperolite_armor_leggings": "铜橄榄护腿",
+  "item.va.copperolite_axe": "铜橄榄斧",
+  "item.va.copperolite_hoe": "铜橄榄锄头",
```

```suggestion
  "item.va.copperolite_hoe": "铜橄榄锄",
```
这个copperolite是铜和moerolite的合金吗
这两种材料分别都长什么样

---

是这俩的合金，偏绿色，配合翻译工具给出的橄榄色结合翻译为铜橄榄